### PR TITLE
Display frontend version

### LINKED ISSUE #3104: Set a version for the frontend code
In the footer we show the backend version, which is useful for verifying that you're using the version you expect to. But the frontend code should have a version too to allow similar verification (and in the future notification for users to reload for newer version).

We can use [git-revision-webpack-plugin](https://www.npmjs.com/package/git-revision-webpack-plugin) or similar.

### PR DESCRIPTION
**Pull request by rauchy**
Closes #3104 .

### CODE REVIEW

File: client/app/components/Footer.jsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ export function Footer({ clientConfig, currentUser }) {
 
   return (
     <div id="footer">
-      <a href="https://redash.io">Redash</a> {version}
+      <a href="https://redash.io">Redash</a> {version} ({COMMIT})
```
Comment (arikfr) [inline client/app/components/Footer.jsx]: To avoid confusion I would name them `backendVersion` and `frontendVersion`.

**Comment by arikfr** [comment]
You probably need to update the jest snapshot.

**Comment by arikfr** [comment]
webpack fails to run inside a container, because: `git describe --always` fails ("not a git repo") :-( Which makes sense... it's not a git repo when docker build runs webpack.

I guess we need an alternative approach here that won't involve using `git` directly.

**Comment by rauchy** [comment]
@arikfr what do you think about this approach: when containers are created, write the latest SHA at creation into a VERSION file and read from it during webpack's build. If the file is not found, revert to using `git-revision-webpack-plugin` which will handle anyone cloning the repo directly.

If that makes sense, I just need a pointer to the scripts that create the different containers...

**Pull request merged**